extensions/android/BUILD.gn

```diff
@@ -0,0 +1,22 @@
+import("//build/config/android/rules.gni")
+
+android_library("xwalk_core_extensions_java") {
+  deps = [
+    "//base:base_java",
+    "//content/public/android:content_java",
+  ]
+  java_files = [
+    "java/src/org/xwalk/core/internal/extensions/XWalkExtensionAndroid.java",
+    "java/src/org/xwalk/core/internal/extensions/XWalkNativeExtensionLoaderAndroid.java",
+  ]
+}
+
+generate_jni("xwalk_core_extensions_native_jni") {
+  jni_package = rebase_path(
```

Comment:
Why complicate this assignment with this `rebase_path` call. Using something as simple as "xwalk" or "extensions" works just fine (again, look at how `generate_jni` is implemented).